src/index.html

```diff
@@ -6,11 +6,43 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
+   
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="/src/style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="#" 
+          method="post" 
+          data-qa="big" 
+          class="Form1">
```

Comment:
```suggestion
    <form 
      action="#" 
      method="post" 
      data-qa="big" 
      class="Form1"
    >
```
Форматируй теги таким образом